### DISCUSSION TITLE
Pcx file format support, wanted ?

### DISCUSSION THREAD

**Discussion by ynse01** [SixLabors/ImageSharp#2497] [Ideas] [answered]
I would like to work on adding support for the [pcx](https://en.wikipedia.org/wiki/PCX) file format to the ImageSharp library.

Would you guys be interested in a PR for the same ?

**Comment by JimBobSquarePants** [comment]
I'm a sucker for these old formats so If you're sure you have the time, I'm happy to accept it. 

As an experiment it would be cool if you could try building the codec initially separate from the main source. I'm confident the new APIs allow for simple extension but it would be good to know if I missed something important during the refactor.

**Reply by antonfirsov** [reply]
@ynse01 is PCX still relevant and if yes I wonder what are real-life use-cases of the format?

**Reply by ynse01** [reply]
For me, this would be just for the fun of historic research into the format itself. I have no real-life use for it myself.

**Reply by JimBobSquarePants** [reply, marked as answer]
I think then in that case it would be best done as a separate project. That’s a good test of our extensibility anyway.
